src/reanimated2/NativeMethods.ts

```diff
@@ -21,36 +19,42 @@ export interface MeasuredDimensions {
   pageY: number;
 }
 
-const isNativeIndefined = shouldBeUseWeb();
+const isNativeUndefined = shouldBeUseWeb();
```

Comment:
I wish we could come up with proper names here 😄

Reply:
Yup, I was trying to avoid making too many subjective decisions, but I agree!